.github/workflows/server-tests.yml

```
@@ -136,9 +136,10 @@ jobs:
           uses: actions/download-artifact@v7
 
         - name: Upload coverage data
+          if: secrets.CODECOV_TOKEN != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)